app/Services/Bibles/BibleFilesetService.php

```diff
@@ -95,6 +90,12 @@ public static function getFilesetFromValidFilesets(
             $available_filesets[] = $partial_fileset;
         }
 
+        $complete_fileset = $valid_filesets->where('set_type_code', $type)->where('set_size_code', 'C')->first();
```

Comment:
@bradflood I have changed the position of this constraint because the priority will be for the filesets that match according to the testament.